packages/rest/src/router/routing-table.ts

```diff
@@ -156,11 +156,13 @@ export class RoutingTable {
     const paths: PathObject = {};
 
     for (const route of this._router.list()) {
-      if (!paths[route.path]) {
-        paths[route.path] = {};
-      }
+      if (!route.spec['x-internal']) {
```

Comment:
Please use reverse logic + early return to make the code easier to follow and less nested.

```ts
if (route.spec['x-internal']) continue;
```